src/video/windows/SDL_windowsopengl.c

```diff
@@ -548,6 +551,10 @@ static int WIN_GL_ChoosePixelFormatARB(_THIS, int *iAttribs, float *fAttribs)
                                                     &matching);
         }
 
+        //Check whether we actually got an SRGB capable buffer
```

Comment:
```suggestion
        /* Check whether we actually got an SRGB capable buffer */
```
The other comments also are c style.

Reply:
Thanks sorry, I'm used to C++